app/src/main/java/com/example/taskyy/ui/LoginScreen.kt

```diff
@@ -175,13 +174,13 @@ fun CreateHidePasswordToggle(isPasswordVisible: Boolean, onEvent: () -> Unit) {
 }
 
 @Composable
-fun CreateLoginButton(onLoginEvent: ((LoginEvent) -> Unit)?, onRegisterEvent: ((RegisterEvent) -> Unit)?, navController: NavController, text: String) {
+fun CreateLoginButton(onClick: () -> Unit, text: String) {
     Button(
         onClick = {
-                  if(onLoginEvent != null) {
-                    //onLoginEvent(LoginEvent.OnLoginClick)
+                  if(text == "LOG IN") {
+                      onClick()
                   } else {
-                     //onRegisterEvent(RegisterEvent.OnGetStartedClick)
+                     onClick()
                   }
```

Comment:
Few things:
1. First, I'd rename the button to something like `TaskyActionButton`, so it's more generic and shows that it's intended to be used for more than just log in.
2. In your `onClick` lambda, all you need to call is the `onClick()` lambda from your parameters. No need for any if-checks, since the screen that adds this button already knows what it is for. Just the button doesn't need to know that.